addons/repair/stringtable.xml

```diff
@@ -2224,11 +2224,11 @@
             <Russian>Порог починки колеса</Russian>
         </Key>
         <Key ID="STR_ACE_Repair_patchWheelMaximumRepair_Description">
-            <English>Maximum level to which a wheel can be patched.</English>
+            <English>Maximum damage to which a wheel can be patched.\n0% means all damage can be repaired.</English>
             <Japanese>タイヤを補修できる最大の度合い。</Japanese>
-            <Polish>Maksymalny poziom, do którego koło może zostać załatane.</Polish>
+            <Polish>Maksymalny poziom, do którego koło może zostać załatane.\n0% oznacza że każde uszkodzenia mogą być naprawione.</Polish>
             <Italian>Livello di integrità massimo di una ruota rattoppata.</Italian>
-            <German>Maximales Level, bis zu dem ein Rad geflickt werden kann.</German>
+            <German>Maximales Level, bis zu dem ein Rad geflickt werden kann.\n0% bedeutet, dass das Rad vollständig repariert werden kann.</German>
             <Korean>바퀴를 수리할 수 있는 최대 레벨입니다.</Korean>
             <French>Niveau maximum de rafistolage d'une roue.</French>
```

Comment:
> Niveau maximum de dégâts jusqu'à laquelle une roue peut être réparée.
> \n %0 veux dire que tous les dommages peuvent être réparés.

asked someone i know for a french translation, but i dont know any french, so if someone who does could double check it before i put it in :)

Reply:
`\n0% signifie que la roue peut être reparée entièrement.`

Reply:
I agree with the changes to the french translation.